kitchensink/runtime/src/lib.rs

```diff
@@ -285,16 +285,19 @@ impl Get<u32> for SubMetadataLen {
 
 type Rand = [u8; 32];
 type Round = u64;
+type Sig = [u8; 64];
```

Comment:
You could use the types here instead: https://github.com/ideal-lab5/idn-sdk/blob/c7aa4779b2debf94182b974eaf2331d62994f43a/pallets/randomness-beacon/src/types.rs#L24

Specifically, there are types for RoundNumber and OpaqueSignature that would be useful here.